ufmt/tests/core.py

```diff
@@ -55,6 +56,19 @@ def func(arg: str = "default") -> bool:
 '''
 
 
+def gen_pyproject_toml(**options):
+    def format(value):
+        if isinstance(value, bool):
+            return str(value).lower()
+        else:
+            return value
+
+    content = "[tool.black]\n\n"
+    for name, value in options.items():
+        content += f"{name} = {format(value)}\n"
+    return content
```

Comment:
Can you rebase this onto main, and replace this with some default config values in FAKE_CONFIG?